app/Http/Controllers/ShopController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Model\Product;
+
+class ShopController extends Controller
+{
+    public function index()
+    {
+
+        $products = Product::paginate(6)->withPath('products');
+
+        return view('user.products', [ 'products' => $products ]);
```

Comment:
if 1 object , we use compact in laravel. =)))
return view('user.products', compact('products'));

Reply:
Fixed